components/cluster/pcs/Makefile

```diff
@@ -26,6 +27,87 @@ COMPONENT_ARCHIVE_URL= https://github.com/ClusterLabs/$(COMPONENT_NAME)/archive/
 COMPONENT_LICENSE= GPLv2
 COMPONENT_SUMMARY= Pacemaker command line interface and GUI.
 
+#
+# The build process tries to use wget to download an archive without even a
+# check for the correct hash.  In order to allow the build to work without
+# networking, we download it first.  This will need to be updated if the
+# version or URL in "pcs/snmp/Makefile" changes.
+#
```

Comment:
pcs/snmp/Makefule? What has snmp to do with it?

Reply:
Within the source we are building, there is a `pcs/snmp/Makefile`.  It is what drives the download of `pyagent`. See: https://github.com/ClusterLabs/pcs/blob/pcs-0.9/pcs/snmp/Makefile#L1-L2

Reply:
Thank you for the explanation. It wasn't obvious for me.

Reply:
You're welcome!